governance/communications/call_to_action.md

```diff
@@ -0,0 +1,14 @@
+# Call to Action template
+
+## Security Enthusiasts!
+CNCF Security Tag is looking to embark on another tech security adventure!  [2 sentences explanation of what the thing is].
+[background about why this exists, about 2-3 sentences].  We'd like some help.
+ 
+## We want you!
+This is where you (yes you!) come in.  We're looking for community members and colleagues [review/contribute] to this [what the thing is called], tracked under [issue number and link to issue], so that we can provide the cloud native community with [what the intent is]!
+[what this means for the community]. We hope you can join us on this significant contribution opportunity. [If needed by a specific date, say so here].
+
+## Ready to dive in?  
+Comment on the issue[relink the issue] (#XXX) and join the [link to the slack channel] Slack channel!
+[Relink everything again here with what it is and the URLs blown out (not embedded)]
```

Comment:
How about putting a line space between these and have a Links heading

Reply:
I didn't get this, can you pls suggest what can be added?